server/src/test/java/io/crate/integrationtests/DDLIntegrationTest.java

```diff
@@ -909,8 +909,8 @@ public void testAddGeneratedColumnToTableWithExistingGeneratedColumns() throws E
                                  "{\"dynamic\":\"strict\"," +
                                  "\"_meta\":{" +
                                  "\"generated_columns\":{" +
-                                 "\"added\":\"date_trunc('day', ts)\"," +
-                                 "\"day\":\"date_trunc('day', ts)\"}}," +
+                                 "\"day\":\"date_trunc('day', ts)\"," +
```

Comment:
I intentionally added each type of constraint in `test_add_column_all_supported_configs_applied_to_altered_table` in `CREATE TABLE`  in order to align `ADD COLUMN` with existing data structures, checked structures in runtime. I think new revision is correct as "added" column goes after existing "day" column.